src/test/com/codingnomads/betty/logic/services/ProbabilityToWinServiceTests.java

```diff
@@ -0,0 +1,52 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.TeamProbabilityToWin;
+import com.codingnomads.betty.logic.models.TeamSentimentScore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProbabilityToWinServiceTests {
+
+    @Test(expected = NullPointerException.class)
+    public void ifTeamSentimentScoreIsNull_throwANullPointerException() {
+        TeamSentimentScore teamSentimentScore = new TeamSentimentScore();
+        teamSentimentScore.setScore(null);
+
+        ProbabilityToWinService service = new ProbabilityToWinService();
+        service.getProbabilityToWinFromSentimentAnalysis(teamSentimentScore);
+
+    }
+
+    @Test
+    public void givenAPositiveTeamSentimentScore_aTeamProbabilityToWinBetween0And100IsReturned() {
+        TeamSentimentScore givenTeamSentiment = new TeamSentimentScore();
+        givenTeamSentiment.setScore(227);
+
+        ProbabilityToWinService service = new ProbabilityToWinService();
+
+        assertEquals(givenTeamSentiment.getScore() / 100,
```

Comment:
I recommend to use the AssertJ assertions , the output are easier to read and faster to solve errors.

```
double probabilityToWin = service.getProbabilityToWinFromSentimentAnalysis(givenTeamSentiment);
assertThat(probabilityToWin.getProbabilityToWin()).isEqualTo(givenTeamSentiment.getScore() / 100);
```

Reply:
![tenor-250562080](https://user-images.githubusercontent.com/32600398/56577859-af615180-65fe-11e9-9653-333db16fc18e.gif)
Done!